model/prelude.sail

```diff
@@ -191,114 +248,112 @@ val EXTZ : forall 'n 'm, 'm >= 'n. (implicit('m), bits('n)) -> bits('m)
 function EXTS(m, v) = sail_sign_extend(v, m)
 function EXTZ(m, v) = sail_zero_extend(v, m)
 
-val zeros_implicit : forall 'n, 'n >= 0 . implicit('n) -> bits('n)
-function zeros_implicit (n) = sail_zeros(n)
+val zeros_implicit : forall 'n, 'n >= 0. implicit('n) -> bits('n)
+function zeros_implicit n = sail_zeros(n)
```

Comment:
Hm, this is going to apply to a lot of functions, including all execute clauses

Reply:
I can make the function argument handling configurable.

Reply:
We do seem to be a bit inconsistent just looking at the next few lines - I don't mind whether parens should be required or always removed, but we should be consistent.